pages/implementation/logs.mdx

```diff
@@ -38,6 +38,30 @@ Avo.initAvo(...)
 Avo.avoLogger = MyAvoLogger()
 ```
 
+### JavaScript
```

Comment:
Before we go into the code examples I think we should say: "Custom loggers are currently only supported in Swift and Javascript. Reach out to us at X if you want access to custom loggers in your programming languages

Reply:
Also: Add title like this above Swift as well

Reply:
👍